src/ui/components/shared/UserSettingsModal/panels/Experimental.tsx

```diff
@@ -2,22 +2,7 @@ import { config } from "shared/user-data/GraphQL/config";
 import { PreferencesKey } from "shared/user-data/GraphQL/types";
 import { BooleanPreference } from "ui/components/shared/UserSettingsModal/components/BooleanPreference";
 
-export const PREFERENCES: PreferencesKey[] = [
-  "feature_columnBreakpoints",
```

Comment:
removing column breakpoints because I'm pretty sure they were removed in the source rewrite.

Reply:
This branch is stale.